client/layout/masterbar/woo-core-profiler.tsx

```diff
@@ -3,6 +3,9 @@ import { Button } from '@wordpress/components';
 import { getQueryArg } from '@wordpress/url';
 import { localize } from 'i18n-calypso';
 import { Fragment } from 'react';
+// @ts-expect-error Missing definition
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+import validUrl from 'valid-url';
```

Comment:
Maybe import only what you need 
```suggestion
import { isWebUrl } from 'valid-url';
```